packages/styles/scss/components/feature-card-block-large/_feature-card-block-large.scss

```diff
@@ -134,6 +134,10 @@ $fcb-breakpoint-up--lg: map-get(
       @include carbon--type-style('productive-heading-05');
     }
 
+    .#{$prefix}--card__footer {
+      flex-direction: row-reverse;
+    }
```

Comment:
Thank you for trying to fix this @IgnacioBecerra! Seems that we need deeper look at why React is working vs. why Web Components doesn't. The style rule here is reused between React and Web Components, and the change here seems to have adverse effect to React (as Percy shows).

Reply:
I pushed a different Web-Component specific change that should fix this. However, the same changes are seen through Percy. From what I can see though, isn't the only change the arrow moving to the right, as it should be?

Reply:
@IgnacioBecerra OK given we have web-component specific change the fixes the problem, sounds that we don't need anything here? Or am I missing something? If it the case, could you please elaborate more on what you mean by:

> However, the same changes are seen through Percy. From what I can see though, isn't the only change the arrow moving to the right, as it should be?

Thanks!

Reply:
Yeah, I think the issue is fixed on both React and Web Components. As for the comment, I was a little confused as Percy didn't show any adverse effect to React to my knowledge in the original commit. Please let me know if there's a Percy change I'm not noticing, but I think we're good now! 

Thanks Akira!